src/main/java/com/conversions/BinaryToGray.java

```diff
@@ -0,0 +1,30 @@
+package src.main.java.com.conversions;
+
+public class BinaryToGray
+{
+	/*	This method convert the binary number into gray code
+		@param binaryno need to convert binary number into gray code
```

Comment:
`@param` name is incorrect, should be `binarycode`

```suggestion
		@param binarycode need to convert binary number into gray code
```